tests/installation/ipxe_install.pm

```diff
@@ -131,26 +131,43 @@ END_BOOTSCRIPT
     diag "$response->{status} $response->{reason}\n";
 }
 
+
 sub run {
     my $self = shift;
 
     poweroff_host;
 
-    #virtualization tests use a static ipxe configuration file
-    set_bootscript unless get_var('VIRT_AUTOTEST');
+    #virtualization tests use a static ipxe configuration file in O3
+    set_bootscript unless get_var('VIRT_AUTOTEST') && is_tumbleweed;
 
     set_pxe_boot;
 
     poweron_host;
 
+    select_console 'sol', await_console => 0;
+
+    #use ipxe bootloader in O3
+    #it is static menu and choose the TW entry to start installation
```

Comment:
iPXE module already supports "boot scrip customization", why virt test use static menu ?

Reply:
I think the power of iPXE lies in "customized boot script" and "iPXE command line", but virt test looks backward and use static menu instead.

Reply:
It is because the special infra restriction in O3. There is no independent dhcp server and htttp server. The static iPXE menu is a tradeoff.